applications/CompressiblePotentialFlowApplication/python_scripts/apply_bc_process_fparts.py

```diff
@@ -0,0 +1,113 @@
+from KratosMultiphysics import *
+from KratosMultiphysics.TestPotentialApplication import *
+
+import KratosMultiphysics
+
+#import KratosMultiphysics.FluidDynamicsApplication
+#import KratosMultiphysics.ConvectionDiffusionApplication 
+#import KratosMultiphysics.FullPotentialFlowApplication
+from array import array
+from numpy import *
```

Comment:
I thought we were supposed to avoid using numpy because it is an external dependency